MediaBrowser.Common/Json/Converters/JsonGuidConverter.cs

```diff
@@ -11,7 +11,11 @@ public class JsonGuidConverter : JsonConverter<Guid>
     {
         /// <inheritdoc />
         public override Guid Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
-            => new Guid(reader.GetString());
+        {
+            var guidStr = reader.GetString();
+
+            return guidStr == null ? Guid.Empty : new Guid(guidStr);
+        }
```

Comment:
```suggestion
       => new Guid(reader.GetString() ?? Guid.Empty);
```

Reply:
There is no `Guid(Guid)` constructor
https://docs.microsoft.com/en-us/dotnet/api/system.guid.-ctor?view=net-5.0

Reply:
Even if there was you'd be doing an allocation just to save one line of code

Reply:
`Guid` is a struct, so not really

Reply:
I'm opting to keep the code as is